spec/features/user_views_all_tattoos_spec.rb

```diff
@@ -23,4 +24,28 @@
     expect(page).to have_content(tattoo4.title)
   end
 
+  scenario 'visitor views only 6 tattoos per page' do
+    tattoo = FactoryGirl.create(:tattoo)
+    tattoo2 = FactoryGirl.create(:tattoo)
+    tattoo3 = FactoryGirl.create(:tattoo)
+    tattoo4 = FactoryGirl.create(:tattoo)
+    tattoo5 = FactoryGirl.create(:tattoo)
+    tattoo6 = FactoryGirl.create(:tattoo)
+    tattoo7 = FactoryGirl.create(:tattoo)
+    tattoo8 = FactoryGirl.create(:tattoo)
```

Comment:
I would strongly suggest creating these factories like so:

```
FactoryGirl.create(:tattoo, title: "Tattoo1")
... # like above
```

and this then allows you to write your test assertions like so:

```
expect(page).to have_content("Tattoo1")
```

The reason for this is if somehow the `tattoo` ruby objects you are creating in the tests somehow start creating tattoo objects with `nil` values for their titles, now you'd have test assertions that are:

```
expect(page).to have_content(nil)
```

which is asserting nothing. It's better to be a bit more hard coded in some tests.